.github/workflows/ci.yaml

```diff
@@ -12,11 +12,11 @@ jobs:
           { os: ubuntu-latest, preset: x64-linux-gcc-dynamic },
           { os: ubuntu-latest, preset: x86-linux-gcc-dynamic },
           { os: ubuntu-latest, preset: x64-linux-clang-dynamic },
-          { os: ubuntu-latest, preset: x86-linux-clang-dynamic },
```

Comment:
This was removed from `.github/ci.yaml` in https://github.com/msvetkin/cpp-project-template/commit/5efee87aab8d0d486c7f80d3aa1ce9862c98800c, so I removed it here in `.github/workflows/ci.yaml` as well. It fails due to missing `libc++abi` on x86